hal/src/common/gpio/v2/dynpin.rs

```diff
@@ -412,11 +418,11 @@ impl DynPin {
     }
     #[inline]
     fn _is_low(&self) -> Result<bool, Error> {
-        Ok(self._read()? == false)
+        Ok(!(self._read()?))
```

Comment:
This doesn't really feel like an improvement...
But to get rid of the warning I suppose a clippy attributes would have to be added?

Reply:
I agree that it's not really an improvement. But I also don't really care enough to fight it.